app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::V1::ItemsController < ApplicationController
+  def create
+    return render json: Error.missing_params, status: :bad_request if missing_params?
```

Comment:
Error is a custom error poro I built to make my error messages robust and custom